config/core/configmaps/autoscaler.yaml

```diff
@@ -165,8 +165,8 @@ data:
     scale-to-zero-pod-retention-period: "0s"
 
     # pod-autoscaler-class specifies the default pod autoscaler class
-    # that should be used if none is specified. If omitted, the Knative
-    # Horizontal Pod Autoscaler (KPA) is used by default.
+    # that should be used if none is specified. If omitted, the 
+    # Knative Pod Autoscaler (KPA) is used by default.
```

Comment:
Hi @HongyuHe - could you [run `./hack/update-checksums.sh`](https://github.com/knative/serving/blob/main/DEVELOPMENT.md#iterating) please? this should fix the build error :)

Reply:
Thank you @julz 
Also rebased.